dataeng/jobs/analytics/AnalyticsEmailOptin.groovy

```diff
@@ -61,8 +63,6 @@ class AnalyticsEmailOptin {
                         'Branch from the edx-analytics-exporter repository. For tags use tags/[tag-name]. Should be environment/production.')
                 stringParam('PLATFORM_BRANCH','zafft/analytics-exporter-settings-hotfix',
                         'Branch from the edx-platform repository. For tags use tags/[tag-name]. Should be release.')
-                stringParam('SECURE_BRANCH','release',
-                        'Branch from the analytics-secure repository, where the configuration settings reside. For tags use tags/[tag-name]')
```

Comment:
Note that removing this here may result in a different default value, depending on what ANALYTICS_SECURE_RELEASE is set to.

Reply:
ANALYTICS_SECURE_RELEASE is set to 'release' as well.  It's set programmatically via:

https://github.com/edx-ops/analytics-secure/blob/master/ansible/analytics-jenkins-override-vars.yml#L102-L103

I suspect setting this variable globally is more confusing that its worth (adds another questionable layer of abstraction), but I could be missing context, and changing it would creep the scope of this PR even more than it has already.